app/gui/view/component-browser/searcher-list-panel/src/column_grid.rs

```diff
@@ -0,0 +1,234 @@
+//! Wrapper around multiple [`component_group::View`] that provides a layout where the
+//! [`component_group::View`] are stacked in three columns. Designed for use in the sections of a
+//! Component Browser Panel.
+
+use ensogl_core::display::shape::*;
+use ensogl_core::prelude::*;
+
+use enso_frp as frp;
+use ensogl_core::application::frp::API;
+use ensogl_core::application::Application;
+use ensogl_core::data::color;
+use ensogl_core::define_endpoints_2;
+use ensogl_core::display;
+use ensogl_core::display::style;
+use ensogl_gui_component::component;
+use ensogl_list_view as list_view;
+use ide_view_component_group as component_group;
+use ide_view_component_group::Layers;
+use ordered_float::OrderedFloat;
+
+
+// =============
+// === Model ===
+// =============
+
+/// Contains a [`AnyModelProvider`] with a label. Can be used to populate a
+/// [`component_group::View`].
+#[derive(Clone, Debug, Default)]
+pub struct LabeledAnyModelProvider {
+    /// Label of the data provided to be used as a header of the list.
+    pub label:   String,
+    /// Content to be used to populate a list.
+    pub content: list_view::entry::AnyModelProvider<component_group::Entry>,
+}
+
+
+
+// =============
+// === Model ===
+// =============
+
+/// The Model of the [`ColumnGrid`] component.
+#[derive(Clone, Debug, CloneRef)]
+pub struct Model {
+    app:            Application,
+    display_object: display::object::Instance,
+    content:        Rc<RefCell<Vec<component_group::View>>>,
+    size:           Rc<Cell<Vector2>>,
+}
+
+impl Model {
+    fn new(app: &Application) -> Self {
+        let logger = Logger::new("ColumnGrid");
+        let app = app.clone_ref();
+        let display_object = display::object::Instance::new(&logger);
+
```

Comment:
Why this nl? I was asking about similar thing in previous review. Could you please check other places in the code too?